src/handlers/strategies/serveFile.ts

```diff
@@ -42,6 +43,64 @@ function getStatusCode(request: Request, objectHasBody: boolean): number {
   return 304;
 }
 
+/**
+ * Fetch an object from a R2 bucket with retries
+ *  The bindings _might_ have retries, if so this just adds
+ *  a little bit more resiliency
+ * @param bucket The {@link R2Bucket} to read from
+ * @param key Object key
+ * @param options Conditional headers, etc.
+ */
+async function r2GetWithRetries(
+  bucket: R2Bucket,
+  key: string,
+  options?: R2GetOptions
+): Promise<R2Object | null> {
+  let retriesRemaining = R2_RETRY_LIMIT;
+  while (retriesRemaining > 0) {
+    try {
+      const file = await bucket.get(key, options);
```

Comment:
```suggestion
      return await bucket.get(key, options);
```

Why assignining a const? 🤔